doc/nrf/ug_nrf70_developing_powersave.rst

```diff
@@ -0,0 +1,325 @@
+
+* wakes up to receive DTIM beacons and decode TIM.
+* receives all broadcast or multicast frames after DTIM beacon.
+* retrieves all unicast frames using either PS-POLL or Trigger frames.
+* maintains the Wi-Fi connection by responding to AP's keep alive packet exchange at any point of time.
+
+TWT based power save allows devices to sleep for longer intervals than the DTIM power save.
+It is suitable for devices which have low levels of periodic uplink traffic.
+As the device sleeps for longer duration and does not wake up to receive DTIM beacons, it misses all the multicast or broadcast frames.
+The TWT session is expected to be set up by the application after the network level negotiation, after which it is not expected to receive any multicast or broadcast frames.
+
+Power Save API
+**************
+
+The following shell commands and net management APIs are provided for power save operation.
+See :ref:`wifi_shell_sample` sample for more information.
+
+.. list-table:: Wi-Fi shell subcommands
+   :header-rows: 1
+
+   * - Net management APIs
+     - Command
+     - Description
+     - Expected output
+   * - net_mgmt(NET_REQUEST_WIFI_PS)
+     - wifi ps on
+     - Turn on power save feature
+     - Power Save enabled
+   * - net_mgmt(NET_REQUEST_WIFI_PS)
+     - wifi ps off
+     - Turn off power save feature
+     - Power Save disabled
+   * - net_mgmt(NET_REQUEST_WIFI_PS_MODE)
+     - wifi ps_mode legacy
+     - Config mode as Legacy
+     -
+   * - net_mgmt(NET_REQUEST_WIFI_PS_MODE)
+     - wifi ps_mode wmm
+     - Config mode as WMM
+     -
+   * - net_mgmt(NET_REQUEST_WIFI_TWT)
+     - wifi twt setup 0 0 1 1 0 1 1 1 65 524
+     - | Set up TWT:
+       | TWT interval - 524 ms
+       | TWT wake interval - 65 ms
+     - TWT operation TWT setup with dg - 1, flow_id - 1 requested
+   * - net_mgmt(NET_REQUEST_WIFI_TWT)
+     - wifi twt teardown 0 0 1 1
+     - Tear down TWT session
+     - TWT operation TWT setup with dg - 1, flow_id - 1 requested
+   * - net_mgmt(NET_REQUEST_WIFI_TWT)
+     - wifi twt teardown_all
+     - Tear down all sessions
+     - TWT operation TWT teardown all flows
+
+Power profiling
+***************
+
+The Power Profiler Kit II of Nordic Semiconductor can be used to measure the power consumption of nRF70 series devices in low power mode.
+To measure the power consumption of the nRF70 series device, complete the following steps:
+
+1. Remove the jumper on **P23** (VBAT jumper).
+#. Connect **GND** on PPK2 kit to any of **GND** on the DK.
+   You can use **P21** pin **1** mentioned as  **GND** (-)
+#. Connect the **Vout** on PPK2 to **P23** pin **1** on the DK.
+
+   .. figure:: ../nrf/images/power_profiler2_pc_nrf7002_dk.svg
+      :alt: Typical configuration for measuring power on the DK
+
+      Typical configuration for measuring power on the DK
+
+#. Configure PPK2 as a source meter with 3.7 volts.
+
+   * The image below shows the Power Profiler Kit II output for DTIM wakeup.
```

Comment:
```suggestion
   * The following image shows the Power Profiler Kit II example output for DTIM wakeup.
```
II or 2?

Reply:
I think, its Power Profiler Kit II in full form and PPK2 in abbreviation.
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/app_power_opt.html